modules/input/react/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@workday/canvas-kit-react-input",
+  "version": "0.0.0",
```

Comment:
This version should also match the other modules.

Reply:
@Parker-Ledoux @MackenzieBerliner-Glasser this has to be `0.0.0` before a release is on npm. This instructs yarn to only look at local files. Changing this will fail CI